src/actions/arcActions.ts

```diff
@@ -217,7 +217,7 @@ export async function getDAOData(avatarAddress : string, currentAccountAddress :
         await new Promise((resolve) => {
           eventFetcher.get((err, events) => {
             if (typeof err === 'undefined' && events.length > 0) {
-              genesisProposal.reputationWhenExecuted = Number(web3.fromWei(events[0].args._totalReputation, "ether"));
+              genesisProposal.reputationWhenExecuted = Number(web3.fromWei((events[0].args as any)._totalReputation, "ether"));
```

Comment:
https://github.com/daostack/arc.js/issues/129

Reply:
this has been fixed on master already, please update to the latest master

Reply:
updated: https://github.com/daostack/alchemy/pull/18/commits/767e38aac36777be3736bc59365ba65c66acde49